_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,354 @@
+---
+layout: post
+title: Deploying Laravel to Kubernetes
+date: 2018-02-18 00:00:00
+categories: kubernetes laravel php
+author: "Keith Mifsud"
+author_avatar: "keith_mifsud.jpg"
+image: /blog/deploying-laravel-to-kubernetes/laravel-and-kubernetes.png
+
+---
+
+As most of us know, Laravel is a great framework for developing any PHP application. Whether we need to   prototype a new idea, develop an MVP (Minimum Viable Product) or release a 
+full-fledged enterprise system, Laravel facilitates all of the development 
+tasks and work-flows.
```

Comment:
I think workflow is not hyphenated.

Reply:
Both are correct but I prefer without it not to be hyphenated and will fix it :)